src/main/java/net/minecraftforge/fml/client/SplashProgress.java

```diff
@@ -405,6 +407,7 @@ public void run()
                         Display.sync(100);
                     }
                 }
+                fontRenderer.clear();
```

Comment:
Instead of calling `clear` here you can wrap the above code in a `try-with-resources` block as it is now `Closeable`.